README.md

```diff
@@ -1,10 +1,47 @@
 # What is this working group about?
 
+![banner](~/Desktop/icon.png)
+
+[![standard-readme compliant](https://img.shields.io/badge/standard--readme-OK-green.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
+
+
+## Table of Contents
+- [Introduction](#introduction)
+- [Background](#background)
+- [Meetings](#meetings) 
+- [Contributing](#contributing)
+- [Metrics](#metrics)
+- [Usage](#usage)
+- [Maintainers](#maintainers)
+- [License](#license)
+
+
+
+## Introduction
+
 _Growth, Maturity and Decline_ is a working group focused on the lifecycle of open source projects. We do not valence the stages, but seek to build awareness of those stages among consumers of these metrics. Decline can be ok when a project's utility is waning; or terrible if 20% of the entire internet relies on that project. Context matters, and we are not the arbiters of your context. The "valence" of "good/bad" emerges from use cases and YOUR context.
 
-This is a [CHAOSS](https://chaoss.community) working group, focused on the [Growth-Maturity-Decline][gmd] metrics category. Its aim is to coordinate the definition of the metrics, and the production of software for implementing them, in this area.
+* Need help on more content here ...
+
+## Background
+* need help on content here ...
+
+
+
+## Meetings
+* We meet every Wednesday at 11am CDT (usually 18:00 CET, but beware different switches to Summer time in EU and US, [check your local time](http://www.thetimezoneconverter.com/?t=11am&tz=Chicago&)) in the CHAOSS Zoom room https://unomaha.zoom.us/j/720431288 (Meeting ID for [dial in](https://unomaha.zoom.us/zoomconference?m=DKGo2mmIuOv9xSjphoGZZmYKxr5HFrS9): 720 431 288)
+
+Usually, we try to make decissions mainly during the last meeting of each month,
+so that decisions can be brought when convenient to the monthtly main CHAOSS meeting,
+which is the first meeting of each month. In all the meetings we may have an agenda,
+but random issues can be raised if time allows, after we're done with the agenda.
+Whenever possible, and specially if a decission needs to be made,
+ensure that issues / pull requests about the subject matter were opened some time before the meeting,
+so that anyone had the opportunity of commenting on them, and make up their mind about that subject matter.
+
+**Exceptions** (these days there will be no meeting): none for now
 
-This group meets every Wednesday at 11am CST. [Connection Information and minutes from previous meetings are located here.](./meeting_notes.md)
+* GMD working group meets every Wednesday at 11am CST. [Meeting Notes from previous meetings are available here.](./meeting_notes.md)
```

Comment:
Remove "GMD working group meets every Wednesday at 11am CST", since that tet is already above (the link to the meeting notes is still pertinent).